modules/DMZ/data_kit/process_folder.py

```diff
@@ -0,0 +1,45 @@
+# Import needed libraries
+from PIL import Image  # image libraries and operations
+import os, sys  # directory libraries
+import imghdr  # image file recognition
+import numpy  # numpy arrays
+
+# Sample Paths
+# /mnt/c/Users/Owner/Pictures/SamplePictures/
+# /mnt/c/Users/Tayo/Documents/SE2CS4365/TestImages/
+
+# list of image types
+imgfiles = ['rgb', 'gif', 'pbm', 'pgm', 'ppm', 'tiff', 'rast', 'xbm', 'jpeg', 'jpg', 'bmp', 'png', 'webp', 'exr']
+
+
+# Splitting up this module into smaller ones
+
+# Given a filepath and size, resizeFolder will enter the folder
+# and edit the files to be square, with lxw as sizexsize
+def resizeFolder(path, size):
```

Comment:
So here's the problem, this code is doing too much. It's both targeting every image in a folder and resizing them. What if you want to apply a different transform to the images? Do you create a new version of this function?

I would suggest figuring out a way to break this up into more modular components. Maybe pass a function to it that is a transform to apply to images.